linera-execution/src/lib.rs

```diff
@@ -411,12 +413,37 @@ pub struct RawOutgoingMessage<Message> {
     pub destination: Destination,
     /// Whether the message is authenticated.
     pub authenticated: bool,
-    /// Whether the message can be skipped by the receiver.
-    pub is_skippable: bool,
+    /// The attributes of the message.
+    pub attributes: MessageAttributes,
     /// The message itself.
     pub message: Message,
 }
 
+#[derive(Default, Debug, PartialEq, Eq, Hash, Clone, Serialize, Deserialize, SimpleObject)]
+/// The attributes of a message.
+pub struct MessageAttributes {
+    /// Whether the message can be skipped.
+    pub is_skippable: bool,
+    /// Whether the message was refused by the receiver and is now bouncing back.
+    pub is_bouncing: bool,
```

Comment:
We could make the fields private, so that it's impossible to instantiate one where both are `true`. (Or maybe we will want unskippable bounced messages at some point?)

(Also, we could consider using a bit field and save a few bytes? At least if we add more attributes later.)

Reply:
I'm not sure. There's also the case for an enum